Userland/Libraries/LibWeb/CMakeLists.txt

```diff
@@ -437,6 +437,7 @@ set(SOURCES
     WebAssembly/WebAssemblyTableConstructor.cpp
```

Comment:
Typo in commit message:

This ~~is~~ cannot be done

Reply:
Thanks, fixed!